### PR TITLE
Clean up NFC write tag activity.

### PR DESCRIPTION
**Pull request by maniac103**

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java [hunk 1]
```diff
@@ -131,53 +126,50 @@ public void onResume() {
...
+            return;
+        }
+
         Tag tagFromIntent = intent.getParcelableExtra(NfcAdapter.EXTRA_TAG);
-        String openhabURI = "";
         //do something with tagFromIntent
         Log.d(TAG, "NFC TAG = " + tagFromIntent.toString());
-        Log.d(TAG, "Writing page " + sitemapPage + " to TAG");
-        TextView writeTagMessage = (TextView) findViewById(R.id.write_tag_message);
+        Log.d(TAG, "Writing page " + mSitemapPage + " to TAG");
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java]: `to tag`

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java [hunk 2]
```diff
@@ -187,8 +179,7 @@ public void writeTag(Tag tag, String openhabUri) {
                 writeTagMessage.setText(R.string.info_write_tag_finished);
                 autoCloseActivity();
             } catch (IOException | FormatException e) {
-                if (e.getMessage() != null)
-                    Log.e(TAG, e.getMessage());
+                Log.e(TAG, "Writing to unformatted tag failed: " + e);
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java]: `Log.e(TAG, "Writing to unformatted tag failed", e);`?

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java [hunk 3]
```diff
@@ -206,14 +197,8 @@ public void writeTag(Tag tag, String openhabUri) {
...
-                } catch (IOException e) {
-                    // TODO Auto-generated catch block
-                    if (e != null)
-                        Log.e(TAG, e.getClass().getCanonicalName());
-                    writeTagMessage.setText(R.string.info_write_failed);
-                } catch (FormatException e) {
-                    // TODO Auto-generated catch block
-                    Log.e(TAG, e.getMessage());
+                } catch (IOException | FormatException e) {
+                    Log.e(TAG, "Writing to formatted tag failed: " + e);
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java]: Same as above

**Comment by mueller-ma** [comment]
I don't own a nfc device, so I cannot test it. If you tested it and feel comfortable, I can merge this.

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java (hunk 1 above)
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java]: Done.

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java (hunk 2 above)
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java]: I don't think the stack trace is actually useful here; that's why I chose this way.

**Comment by maniac103** [comment]
> I don't own a nfc device, so I cannot test it. If you tested it and feel comfortable, I can merge this.

Yeah, I just tested this on a Nexus 4, it still works as well as before.

**Pull request merged**